recipes/pyxenon/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pyxenon" %}
+{% set version = "3.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyxenon-{{ version }}.tar.gz
+  sha256: 6840771716974da68d69e5cc24e017698f8f9bcebcfaf3cb5da41555d5ec772b
+  patches:
+    - setupunix.patch  # [unix]
+    - setupwin.patch  # [win]
+
```

Comment:
The `pyxenon` package could be `noarch: python` but windows complains about the `bin/xenon-grpc` bash script and causes the build to fail. So I only copy the operation system specific shell scripts.